omahelsinki/settings.py

```diff
@@ -155,7 +162,9 @@
 }
 
 COMPRESS_PRECOMPILERS = (
-    ('text/x-scss', '%s/node_modules/.bin/node-sass --importer=%s/node_modules/node-sass-tilde-importer {infile} {outfile}' % (BASE_DIR, BASE_DIR)),
+    ('text/x-scss',
+        '%s/node_modules/.bin/node-sass --importer=%s/node_modules/node-sass-tilde-importer {infile} {outfile}'
```

Comment:
I think it's best to leave this line as it is even though it's too long. Splitting it like this reduces readability IMHO.